schemas/rum/error-schema.json

```diff
@@ -47,6 +47,40 @@
               "description": "Stacktrace of the error",
               "readOnly": false
             },
+            "causes": {
+              "type": "array",
+              "description": "Causes of the error",
+              "maxItems": 10,
+              "readOnly": false,
+              "items": {
+                "type": "object",
+                "description": "Properties for one of the error causes",
+                "required": ["message"],
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** like for error, no?‏
```suggestion
                "required": ["message", "source"],
```